ynr/apps/elections/filters.py

```diff
@@ -300,13 +300,28 @@ class CurrentOrFutureBallotFilter(BaseBallotFilter):
     elections
     """
 
+    def has_results_filter(self, queryset, name, value):
```

Comment:
Could you also add a "shortcut" for "has results" please? As per https://github.com/DemocracyClub/yournextrepresentative/blob/bd62d4f14f6977c330496b6c4d39d3c66ca1d015/ynr/apps/elections/filters.py#L341